generate_sharktank.py

```diff
@@ -4,15 +4,11 @@
 # will generate local shark tank folder like this:
 #   /SHARK
 #     /gen_shark_tank
-#       /tflite
-#         /albert_lite_base
-#         /...model_name...
-#       /tf
-#       /pytorch
+#       /albert_lite_base
+#       /...model_name...
 #
 
 import os
-import urllib.request
```

Comment:
would this cause the builders to hang ? All of them hung

Reply:
No, It wouldn't. this request is not used anymore, we use gsutil instead